src/Template/Layout/js/app/components/filter-box.js

```diff
@@ -179,9 +223,29 @@ export default {
             }
         },
 
+        /**
+         * watcher for object type filter
+         * emits a filter-objects event with queryFilter as params
+         *
+         * @param {String} value The filter object type
+         *
+         * @emits Event#filter-objects
+         *
+         * @return {void}
+         */
+        filterType(value) {
+            this.filterType = value;
+            this.queryFilter.filter.type = this.filterType;
+
+            clearTimeout(this.timer);
```

Comment:
A curiosity: what is `this.timer` for?

Reply:
it's a timeout for the text field search, so that the user has the time to type, here actually i just need to clear it